docs/getting_started/install_ez_platform.md

```diff
@@ -317,6 +317,10 @@ Prepare a [virtual host configuration](https://httpd.apache.org/docs/2.4/vhosts/
 You can copy [the example vhost file](https://github.com/ezsystems/developer-documentation/tree/master/code_samples/install/vhost_template/vhost.template)
 to `/etc/apache2/sites-available` as a `.conf` file and modify it to fit your project.
 
+!!! note "Nginx"
+
+    If you are using Nginx instead of Apache, please use  [this example vhost file](https://github.com/ezsystems/developer-documentation/tree/master/code_samples/install/vhost_template/nginx-vhost.template) and modify it to fit your project. Also modify the instructions below accordingly.
```

Comment:
```suggestion
    If you are using Nginx instead of Apache, use [this example vhost file](https://github.com/ezsystems/developer-documentation/tree/master/code_samples/install/vhost_template/nginx-vhost.template) and modify it to fit your project. Also modify the instructions below accordingly.
```